metadata.json

```
@@ -32,6 +32,10 @@
     {
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": ["14.04", "16.04", "17.04", "17.10", "18.04", "18.10"]
+    },
+    {
+      "operatingsystem": "Windows",
+      "operatingsystemrelease": [ "Server 2008", "Server 2008 R2", "Server 2012", "Server 2012 R2", "Server 2016", "Server 2019", "7", "8", "10" ]
```

Our docs page says that we support Windows Server fromm 2008 R2 onwards (see [here](https://docs.datadoghq.com/agent/basic_agent_usage/?tab=agentv6#supported-os-versions)), so maybe we should remove the non-R2 version here?
```suggestion
      "operatingsystemrelease": [ "Server 2008 R2", "Server 2012", "Server 2012 R2", "Server 2016", "Server 2019", "7", "8", "10" ]
```